Ports/stb/package.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port='stb'
+version='af1a5bc352164740c1cc1354942b1c6b72eacb8a'
+files="https://github.com/nothings/stb/archive/${version}.zip stb-${version}.zip e3d0edbecd356506d3d69b87419de2f9d180a98099134c6343177885f6c2cbef"
+auth_type='sha256'
+
+build() {
+    :
+}
+
+install() {
+    :
+}
+
+post_install() {
+    mkdir -p "${SERENITY_INSTALL_ROOT}/usr/include/stb/"
+    run cp -r "./" "${SERENITY_INSTALL_ROOT}/usr/include/stb/"
+}
```

Comment:
Looking at it, we might also want to move those headers to `/usr/local/include`.

Reply:
But why should we move this to `/usr/local/include`? It will clutter that folder for no good reason.
As for using the `install` stage, I'll use that instead.

Reply:
Well, if we keep it in `/usr/include` it just clutters that folder instead. And Ports generally install into `/usr/local`.

Reply:
Done.